group.yml

```diff
@@ -159,78 +159,78 @@ repos:
   rhel-8-baseos-rpms:
     conf:
       baseurl:
-        aarch64: http://rhsm-pulp.corp.redhat.com/content/eus/rhel8/8.4/aarch64/baseos/os/
-        ppc64le: http://rhsm-pulp.corp.redhat.com/content/eus/rhel8/8.4/ppc64le/baseos/os/
-        s390x: http://rhsm-pulp.corp.redhat.com/content/eus/rhel8/8.4/s390x/baseos/os/
-        x86_64: http://rhsm-pulp.corp.redhat.com/content/eus/rhel8/8.4/x86_64/baseos/os/
+        aarch64: http://rhsm-pulp.corp.redhat.com/content/eus/rhel8/8.6/aarch64/baseos/os/
+        ppc64le: http://rhsm-pulp.corp.redhat.com/content/eus/rhel8/8.6/ppc64le/baseos/os/
+        s390x: http://rhsm-pulp.corp.redhat.com/content/eus/rhel8/8.6/s390x/baseos/os/
+        x86_64: http://rhsm-pulp.corp.redhat.com/content/eus/rhel8/8.6/x86_64/baseos/os/
       ci_alignment:
         profiles:
         - el8
         localdev:
           enabled: true
     content_set:
-      default: rhel-8-for-x86_64-baseos-eus-rpms__8_DOT_4
-      aarch64: rhel-8-for-aarch64-baseos-eus-rpms__8_DOT_4
-      ppc64le: rhel-8-for-ppc64le-baseos-eus-rpms__8_DOT_4
-      s390x: rhel-8-for-s390x-baseos-eus-rpms__8_DOT_4
+      default: rhel-8-for-x86_64-baseos-eus-rpms__8_DOT_6
+      aarch64: rhel-8-for-aarch64-baseos-eus-rpms__8_DOT_6
+      ppc64le: rhel-8-for-ppc64le-baseos-eus-rpms__8_DOT_6
+      s390x: rhel-8-for-s390x-baseos-eus-rpms__8_DOT_6
```

Comment:
likewise these won't exist... look back through history for what we used when tracking RHEL main